### ISSUE TITLE
Support for Novation Launchpad mk2?

### ISSUE THREAD

**Issue by LTgekko** [dbkynd/controlcast#31]
Hey, I know the isn't support for my Launchpad but I hope to see it become supported in the future since the sfotware is great and the isn't a lot of programes out their like this and with such a easy to use UI.

I have a Novation Launchpad MK2 but the software of course isn't supported.  It does give me use of 48 pads but it is all mixed up in the software so if I hit 0,0 the software thinks I am hitting 1,5.

I have thought about getting the older version of my Launchpad but thought I'd ask before parting with a fair amount of cash.

**Comment by ruohki** [comment]
I dont think this tool is maintened so i started my own project that already does support the mk2. No source release yet as I need to shape things but if you want to grab it you can do so in the discord I did setup for that purpose: https://discord.gg/4Ys9TRR
